Utils/Dataflow/009_oracleConnector/dbConnection.py

```diff
@@ -119,6 +119,7 @@ def save_query_file(self, qname, src_filename, params=None):
         try:
             f = open(src_filename)
             q = f.read().rstrip().rstrip(';')
+            f.close()
```

Comment:
Yes, that construction is usually better, but I wanted to make the change as small as possible because it's a bit outside of this PR's aim. Since you suggested it - done.

Reply:
No matter how small the correction is -- if it is out of place, it is out of place ;) If a reviewer pay no attention to the change because it is "just one line added, nothing to think about", then shame on the reviewer.

You\`re right saying:
> it's a bit outside of this PR's aim.

To make this change less out-of-place, there are two possible solutions:
* change the PR description (not "query parameters error handling", but "stage (minor?) improvements");
* move it to another PR.

Reply:
Changed the PR's title and description.

Reply:
Maybe "query saving procedure" or "function"? Took me a while to read the new title correctly; still it can be just me and Friday night...

Reply:
I thought about giving the exact function name, but we are also changing the config example, so I decided against that. However, "query saving procedure" looks fine to me. Done.